main/templates/base.html

```diff
@@ -18,6 +18,7 @@
     {% load render_bundle %}
     {% render_bundle 'sentry_client' %}
     {% render_bundle 'style' %}
+    {% render_bundle 'third_party' %}
```

Comment:
Please render this bundle at the end of body. We do not want to block parser or renderer because of this.

Reply:
If I put it in the end, it does not process dynamic carousel by rasing `$ is not defined`.